addon/helpers/href-to.js

```diff
@@ -30,7 +30,10 @@ export default class HrefToHelper extends Helper {
 
   init() {
     super.init();
-    this.router.on('routeDidChange', this.recompute.bind(this));
+
+    if (this.router && this.router.on) { // skip if the router service is mocked
```

Comment:
wouldn't it be better to adjust the router mock instead? 🤔

Reply:
Perhaps, I originally made the change to the mocks in our test suit. There are a fair number of those mocks though, so I thought it best to make the change here. It's a marginal decision I think